SCClassLibrary/SCDoc/SCDocRenderer.sc

```diff
@@ -1043,10 +1043,7 @@ SCDocHTMLRenderer {
 
 		if(doc.isClassDoc) {
 			var moreExamples = "https://sccode.org/tag/class/%".format(doc.klass);
-			stream << (
-				"</br>For community examples, please refer to <a href='"
-				+ moreExamples ++ "'>" ++ moreExamples ++ "</a>."
-			)
+			stream << ("</br><a href='" + moreExamples ++ "'>" ++ moreExamples ++ "</a>")
```

Comment:
This is now lacking the context that these are community examples.
Also, a `br` can not be closed - I don't know if this is used elsewhere in the code base but this is invalid html syntax.

e.g.

```supercollider
var scCodeLink = "https://sccode.org/tag/class/%".format(doc.klass);
stream << ("<br><a href=\"%\">Community examples</a>".format(scCodeLink));
```

would look most clean to me. But please wait for the discussion to resolve before adjusting the implementation ;)